README.md

```diff
@@ -1,100 +0,0 @@
-[![Master Build Status](https://ci.centos.org/buildStatus/icon?subject=master&job=devtools-rh-che-build-che-credentials-master/)](https://ci.centos.org/job/devtools-rh-che-build-che-credentials-master/)
```

Comment:
@Katka92 removal of the README does not look expected

Reply:
Thank you for your catch! I've excluded that change from my commit. Sorry for that.